resource_customizations/external-secrets.io/PushSecret/actions/discovery.lua

```diff
@@ -0,0 +1,3 @@
+actions = {}
+actions["push"] = {["disabled"] = false}
```

Comment:
Hm. Why named "push" instead of "force-sync"?

Reply:
The action it is doing is more of a "push" based on the resource name. The name of the annotation really does not matter, it is just a way to modify the resource so the controller reconciles it. I also think we should avoid sync-related names because there is already an Argo "sync" option and it could be confising.